### PR TITLE
display scheduled appointments,frontend and backend

### PR DESCRIPTION
**Pull request by maka998**

### CODE REVIEW

**Comment by maka998** [comment]
[(3)WA-3.6 As a patient I want to be able to see all scheduled appointments in order to get the information I need.](https://trello.com/c/Wws6gYK4/25-3wa-36-as-a-patient-i-want-to-be-able-to-see-all-scheduled-appointments-in-order-to-get-the-information-i-need)

File: WebApplication/Backend/Repositorys/IRegistrationRepository.cs [hunk 1]
```diff
@@ -14,6 +14,6 @@ public interface IRegistrationRepository
         public String GetPatientId(string idd);
         public bool IsPatientIdValid(string patientId);
         public bool ConfirmEmailUpdate(string patientId);
-        public List<FamilyDoctorDTO> GetAllGeneralPractitionerPhysitians();
+        public List<FamilyDoctorDTO> GetAllGeneralPractitionerPhysitions();
```
Comment (milicasiriski) [inline WebApplication/Backend/Repositorys/IRegistrationRepository.cs]: typo

File: WebApplication/Backend/Services/RegistrationService.cs [hunk 2]
```diff
@@ -48,9 +48,9 @@ public bool ConfirmEmailUpdate(string id)
             return registrationRepository.ConfirmEmailUpdate(id);
         }
 
-        public List<FamilyDoctorDTO> GetAllPhysicians()
+        public List<FamilyDoctorDTO> GetAllPhysitians()
         {
-            return registrationRepository.GetAllGeneralPractitionerPhysitians();
+            return registrationRepository.GetAllGeneralPractitionerPhysitions();
```
Comment (milicasiriski) [inline WebApplication/Backend/Services/RegistrationService.cs]: typo

File: WebApplication/Backend/Repositorys/ProcedureTypeRepository.cs [hunk 3]
```diff
@@ -52,7 +52,7 @@ public ProcedureType GetProcedureTypeBySerialNumber(string serialNumber)
                 return null;
             }
         }
-
+        /*
```
Comment (milicasiriski) [inline WebApplication/Backend/Repositorys/ProcedureTypeRepository.cs:55]: if it's not used or not working delete it

File: WebApplicationTests/AppointmentTests.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+        [Fact]
+        public void Find_Appointments_By_PatientId_Success()
+        {
+            var stubRepository = new Mock<IAppointmentRepository>();
+            bool result;
+            appointments.Add(appointment);
+
+            stubRepository.Setup(m => m.GetAllAppointmentByPatientId(patientIdTrue)).Returns(appointments);
+            WebApplication.Backend.Services.AppointmentsService service = new WebApplication.Backend.Services.AppointmentsService(stubRepository.Object);
```
Comment (milicasiriski) [inline WebApplicationTests/AppointmentTests.cs:30]: import this service so you don't need to reference it like this

File: WebApplicationTests/AppointmentTests.cs [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+
+        [Fact]
+        public void Find_Appointments_By_PatientId_Success()
+        {
+            var stubRepository = new Mock<IAppointmentRepository>();
+            bool result;
+            appointments.Add(appointment);
+
+            stubRepository.Setup(m => m.GetAllAppointmentByPatientId(patientIdTrue)).Returns(appointments);
```
Comment (milicasiriski) [inline WebApplicationTests/AppointmentTests.cs:29]: 💯

File: project-generated-code-backend/Backend/Controller/PhysitianControllers/PhysitianHospitalAccountsController.cs [hunk 6]
```diff
@@ -35,7 +35,7 @@ public List<Patient> GetPatientsByPhysitian()
         {
             return patientAccountsService.getPatientsForPhysitian(loggedPhysitian);
         }
-
+        /*
```
Comment (milicasiriski) [inline project-generated-code-backend/Backend/Controller/PhysitianControllers/PhysitianHospitalAccountsController.cs]: throw Not Implemented Exception if it requires parameter

File: project-generated-code-backend/Backend/Controller/PhysitianControllers/PhysitianScheduleController.cs [hunk 7]
```diff
@@ -22,12 +22,12 @@ public PhysitianScheduleController(Physitian loggedPhysitian)
             this.loggedPhysitian = loggedPhysitian;
             this.physitianScheduleService = new PhysitianScheduleService(loggedPhysitian);
         }
-
+        /*
```
Comment (milicasiriski) [inline project-generated-code-backend/Backend/Controller/PhysitianControllers/PhysitianScheduleController.cs]: throw not implemented exception

File: project-generated-code-backend/Backend/Model/Schedule/Appointment.cs [hunk 8]
```diff
@@ -83,10 +92,11 @@ public override string ToString()
             return "patient: " + patient.FullName + "\nphysitian: " + physitian.FullName + "\ntime interval: "
                 + timeInterval.ToString() + "\nroom: " + room.ToString() + "\nprocedure type: " + procedureType.ToString();
         }
-
+        /*
```
Comment (milicasiriski) [inline project-generated-code-backend/Backend/Model/Schedule/Appointment.cs]: Why is this commented?

File: project-generated-code-backend/Backend/Service/SchedulingService/PhysitianScheduleService.cs [hunk 9]
```diff
@@ -27,7 +27,7 @@ public void NewAppointment(AppointmentDTO appointmentDTO)
         {
             Appointment appointment = new Appointment(appointmentDTO);
             appointmentRepository.Save(appointment);
-        }
+        }/*
```
Comment (milicasiriski) [inline project-generated-code-backend/Backend/Service/SchedulingService/PhysitianScheduleService.cs]: why is this commented?

File: project-generated-code-backend/Backend/Service/SchedulingService/PhysitianScheduleService.cs [hunk 10]
```diff
@@ -75,8 +75,8 @@ public Appointment GetNextAppointmentForPatient(Patient patient)
                 }
             }
             return null;
-        }
-
+        }*/
```
Comment (milicasiriski) [inline project-generated-code-backend/Backend/Service/SchedulingService/PhysitianScheduleService.cs]: same

**Review by milicasiriski — CHANGES_REQUESTED** [review]
Good PR! Try not to comment old code because most of it is used somewhere and could lead to bugs, if something is causing problems tell me so we can solve it without affecting the rest of the app.

**Review by milicasiriski — APPROVED** [review]

File: WebApplication/Backend/Controllers/RegistrationController.cs [hunk 11]
```diff
@@ -93,7 +93,7 @@ private string IdDecryption(string patientId)
         [HttpGet("allPhysitians")]
         public List<FamilyDoctorDTO> GetAllFeedbacks()
         {
-            return registrationService.GetAllPhysicians();
+            return registrationService.GetAllPhysitians();
```
Comment (VladimirBudjen) [inline WebApplication/Backend/Controllers/RegistrationController.cs]: physiCians was good :(

File: WebApplication/Backend/Repositorys/IRegistrationRepository.cs [hunk 12]
```diff
@@ -14,6 +14,6 @@ public interface IRegistrationRepository
         public String GetPatientId(string idd);
         public bool IsPatientIdValid(string patientId);
         public bool ConfirmEmailUpdate(string patientId);
-        public List<FamilyDoctorDTO> GetAllGeneralPractitionerPhysitians();
+        public List<FamilyDoctorDTO> GetAllGeneralPracticePhysitians();
```
Comment (VladimirBudjen) [inline WebApplication/Backend/Repositorys/IRegistrationRepository.cs]: *GetAllGeneralPracticePhysicians

**Review by VladimirBudjen — CHANGES_REQUESTED** [review]
PhysiTian X
PhysiCian ✓

**Review by VladimirBudjen — APPROVED** [review]

**Pull request merged**